stubs/redis/redis/client.pyi

```diff
@@ -568,7 +567,7 @@ class Pipeline(Redis[_StrType], Generic[_StrType]):
     def sscan_iter(self, name: _Key, match: _Key | None = ..., count: int | None = ...) -> Iterator[Any]: ...
     def hscan(self, name: _Key, cursor: int = ..., match: _Key | None = ..., count: int | None = ...) -> Pipeline[_StrType]: ...  # type: ignore[override]
     def hscan_iter(self, name, match: _Key | None = ..., count: int | None = ...) -> Iterator[Any]: ...
-    def zscan(self, name: _Key, cursor: int = ..., match: _Key | None = ..., count: int | None = ..., score_cast_func: Callable[[_StrType], _ScoreCastFuncReturn] = ...) -> Pipeline[_StrType]: ...  # type: ignore[override]
+    def zscan(self, name: _Key, cursor: int = ..., match: _Key | None = ..., count: int | None = ..., score_cast_func: Callable[[_StrType], Any] = ...) -> Pipeline[_StrType]: ...  # type: ignore[override]
```

Comment:
Looks like the stubs are a little out of date here -- it looks like at runtime, this method is now just inherited from the superclass. I guess there was some time in the past when that wasn't true, but I think we can safely just delete this method now:

```suggestion
```

Reply:
I have not taken a look at source (just figured it's inferred as Any anyway and that's enough to fix the issue). I'll go with your suggestion.